src/views/catalog/wizard/components/WizardFooter.tsx

```diff
@@ -59,12 +64,18 @@ export const WizardFooter: React.FC<{ namespace: string }> = ({ namespace }) =>
   const templateNamespace = tabsData?.overview?.templateMetadata?.namespace;
   const loaded = vmContextLoaded && vmCreateLoaded && bootSourceLoaded;
 
-  React.useEffect(() => {
-    if (loaded && startVM !== isBootSourceAvailable) {
+  useEffect(() => {
+    updateTabsData((tabsDataDraft) => {
+      tabsDataDraft.startVM = startVM;
+    });
+  }, [startVM, updateTabsData]);
```

Comment:
same here for this useEffect, would using the update method in the checkbox onChange prop
```
onChange={(checked) => {
  setStartVM(checked);
  updateTabsData((currentTabsData) => {
    return { ...currentTabsData, startVM: checked };
  });
}}
```